### PR TITLE
Add warning message for expected < 5

### PR DESCRIPTION
**Pull request by koenderks**
Add's a warning message that the assumption 'expected >= 5` is violated in the chi-square test. Also updates tests for unused fairness analysis to be consistent with jfa package.

### CODE REVIEW

File: R/auditClassicalBenfordsLaw.R [hunk 1]
```diff
@@ -335,28 +341,34 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...) {
...
         "uniform" = 1 / length(digits)
       )
-      tb[["lb"]] <- rep(".", length(digits))
-      tb[["ub"]] <- rep(".", length(digits))
       tb[["pval"]] <- rep(".", length(digits))
       tb[["bf"]] <- rep(".", length(digits))
+      if (options[["confidenceInterval"]]) {
+        tb[["lb"]] <- rep(".", length(digits))
+        tb[["ub"]] <- rep(".", length(digits))
+      }
```
Comment (vandenman) [inline R/auditClassicalBenfordsLaw.R:349]: You probably left the existing structure intact for now, but just FYI, `tb$setExpectedSize(length(digits))` would have done this for all columns.

**Review by vandenman — CHANGES_REQUESTED** [review]
There are substantial changes in the robustness plot and sequential plot. Do you know why this is?

**Comment by koenderks** [comment]
@vandenman Yes the changes in the robustness plot are intended since the bf is now based on a joint multinomial sampling plan instead of an independent multinomial sampling plan.

File: R/auditClassicalBenfordsLaw.R (hunk 1 above)
Comment (koenderks) [reply R/auditClassicalBenfordsLaw.R:349]: But it does not give the dots (`.`) in the table, right?

image

Comment (vandenman) [reply R/auditClassicalBenfordsLaw.R:349]: Nope, it does not. Perhaps we should allow people to specify a placeholder symbol, @JorisGoosen what do you think? So `tb$setExpectedSize(length(digits), placeholder = ".")`.

**Review by vandenman — APPROVED** [review]
Ah okay, that clears things up, the differences did look a bit extreme.

**Pull request merged**